.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# The AEP maintainers team owns the content of this repository.
+*   @aep-dev/maintainers
```

Comment:
FYI it looks like this group isn't recognized. indentation issue?

Reply:
It was a repo settings issue; the team didn't have access.  Fixed.